runner/docker/Dockerfile.live-base

```diff
@@ -61,10 +60,7 @@ RUN cd ffmpeg && \
                 --disable-doc && \
     make -j$(nproc) && \
     make install && \
-    make distclean
-
-# Copy the compiled FFmpeg binaries to /usr/local
-RUN cp -R /compiled/* /usr/local/
-
-# Clean up APT cache to reduce image size
-RUN apt-get clean && rm -rf /var/lib/apt/lists/*
```

Comment:
docker has incremental build so a cleanup operation like that must be under the same `RUN` where we install apt-get packages